tests/accordion-test.ts

```diff
@@ -0,0 +1,423 @@
+                    { title: "title 5?", body: "body 5", active: false },
+                ]
+            )
+        );
+        const titles = el.querySelectorAll(".acc-title");
+
+        expect(el.hasAttribute("nomultiple")).to.be.true;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(1);
+        const active = el.querySelectorAll(".acc-active");
+        active.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+
+        titles[0].click();
+        titles[1].click();
+        titles[2].click();
+        await el.updateComplete;
+
+        expect(el.querySelectorAll(".acc-active").length).equal(1);
+
+        const activeItems = el.querySelectorAll(".acc-active");
+        activeItems.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+
+        el.removeAttribute("nomultiple");
+        await el.updateComplete;
+
+        titles[2].click();
+        await el.updateComplete;
+        expect(el.querySelectorAll(".acc-active").length).equal(0);
+        expect(el.hasAttribute("nomultiple")).to.be.false;
+
+        const activeItemsAfter = el.querySelectorAll(".acc-active");
+        activeItemsAfter.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+    });
+
+    it("allows multiple items to be open if nomultiple is false then set as true", async () => {
+        const el = await fixture(
+            createAccordionTemplate({ activecls: "acc-active" }, [
+                {
+                    title: "What is Artificial Intelligence?",
+                    body: "body 1",
+                    active: true,
+                },
+                {
+                    title: "What is Machine Learning?",
+                    body: "body 2",
+                    active: false,
+                },
+                { title: "title 3?", body: "body 3", active: true },
+                { title: "title 4?", body: "body 4", active: true },
+                { title: "title 5?", body: "body 5", active: false },
+            ])
+        );
+
+        const titles = el.querySelectorAll(".acc-title");
+        expect(el.hasAttribute("nomultiple")).to.be.false;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(3);
+        const active = el.querySelectorAll(".acc-active");
+        active.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+
+        titles[0].click();
+        titles[1].click();
+        titles[2].click();
+        await el.updateComplete;
+
+        expect(el.querySelectorAll(".acc-active").length).equal(2);
```

Comment:
Which all index are really opened should be need, isnt it. Thats the only way can verify our intentions.